### ISSUE TITLE
Possibility to have different notebooks

### ISSUE THREAD

**Issue by olivattaque** [wallabag/wallabag#2782]
Would be wonderful to have the possibility to create different notebooks.
Example in my case, i'd like to have two notebooks : 
- one for "content" which is interesting articles i keep on different themes
- one for "monitoring" (veille in french) which keep a lot of different actuality news about some personality I like or dislike, or news about society/politics progress
Then you could use wallabag powerful features for each of those notebooks instead of having everything in the same "notebook" which make things pretty useless... With current version you could use a tag to differentiate articles or another instance of wallabag but this is really not practical nor efficient.

That's just my use case but I think a lot of people could find a useful application of this feature which I haven't seen in wallabag or the github issues (sorry if my post is a duplicate).

Thanks for your work !

**Comment by j0k3r** [comment]
As of now, I'll recommend to use tag for that.
Since this a rare case that can be solved by one existing function, I don't think we'll implement these _notebooks_ (which can be more appropriate renamed _groups_) in the short future.
But thanks for the idea.

**Comment by olivattaque** [comment]
Yes as I said in my first post but it's clearly not an option, fact is i'm not using wallabag because of this, let me be more specific :
- There is no way to search for unread articles with a specific tag (Filters are not available when in a tag list). Even if there was one, there is no possibility to save this search. So when I want to read articles from one of my notebook/group I would have to do a search everytime ? No way. Same problematic goes with favorite, archive etc...
- Everytime I want to add an article, I have to write the name of the tag instead of chosing from pre-existing notebook/group

Not being able to access want I want doesn't solve my problem. I understand that's a lot of work (as this maybe should have be designed from the start) but I think we should at least be able to do advanced search (by tag and state) and save those so we can access them from the left menu, I could open a new issue for it if you want.

Sorry for my trouble, thanks for your work :p

**Comment by nicosomb** [comment]
Is that a duplicate of https://github.com/wallabag/wallabag/issues/2229 ?
